src/subsystems/odometry/odometry_base.cpp

```diff
@@ -26,6 +26,7 @@ int OdometryBase::background_task(void *ptr) {
     obj.mut.lock();
     obj.update();
     obj.mut.unlock();
+    vexDelay(5);
```

Comment:
I think this is a good thing to add to keep from just eating cpu. but to make sure we're not breaking anything can you try like driving tank odom forward and back with the 5ms delay and without and see  if theres any significant difference?

Reply:
tried and there is no difference at all

Reply:
sick